x-pack/plugins/uptime/public/components/overview/alerts/translations.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const TlsTranslations = {
+  criteriaAriaLabel: i18n.translate('xpack.uptime.alerts.tls.criteriaExpression.ariaLabel', {
+    defaultMessage:
+      'An expression displaying the criteria for monitor that are watched by this alert',
+  }),
+  criteriaDescription: i18n.translate('xpack.uptime.alerts.tls.criteriaExpression.description', {
+    defaultMessage: 'when',
+    description:
+      'The context of this `when` is in the conditional sense, like "when there are three cookies, eat them all".',
+  }),
+  criteriaValue: i18n.translate('xpack.uptime.alerts.tls.criteriaExpression.value', {
+    defaultMessage: 'any monitor',
+  }),
+  expirationAriaLabel: i18n.translate('xpack.uptime.alerts.tls.expirationExpression.ariaLabel', {
+    defaultMessage:
+      'An expression displaying the threshold that will trigger the TLS alert for certificate expiration',
+  }),
+  expirationDescription: i18n.translate(
+    'xpack.uptime.alerts.tls.expirationExpression.description',
+    {
+      defaultMessage: 'has a certificate expiring within',
+    }
+  ),
+  expirationValue: (value?: number) =>
+    i18n.translate('xpack.uptime.alerts.tls.expirationExpression.value', {
+      defaultMessage: '{value} days',
+      values: { value },
+    }),
+  ageAriaLabel: i18n.translate('xpack.uptime.alerts.tls.ageExpression.ariaLabel', {
+    defaultMessage:
+      'An expressing displaying the threshold that will trigger the TLS alert for old certificates',
+  }),
+  ageDescription: i18n.translate('xpack.uptime.alerts.tls.ageExpression.description', {
+    defaultMessage: 'or older than',
+  }),
+  ageValue: (value?: number) =>
+    i18n.translate('xpack.uptime.alerts.tls.ageExpression.value', {
+      defaultMessage: '{value} days',
+      values: { value },
+    }),
+};
+
+export const ToggleFlyoutTranslations = {
+  toggleButtonAriaLabel: i18n.translate('xpack.uptime.alertsPopover.toggleButton.ariaLabel', {
+    defaultMessage: 'Open alert context menu',
+  }),
+  openAlertContextPanelAriaLabel: i18n.translate('xpack.uptime.openAlertContextPanel.ariaLabel', {
+    defaultMessage: 'Open the alert context panel so you can choose an alert type',
+  }),
+  openAlertContextPanelLabel: i18n.translate('xpack.uptime.openAlertContextPanel.label', {
+    defaultMessage: 'Create alert',
+  }),
+  toggleTlsAriaLabel: i18n.translate('xpack.uptime.toggleTlsAlertButton.ariaLabel', {
+    defaultMessage: 'Open TLS alert flyout',
+  }),
+  toggleTlsContent: i18n.translate('xpack.uptime.toggleTlsAlertButton.content', {
+    defaultMessage: 'Create TLS alert',
```

Comment:
i think this should be just "TLS alert".

Reply:
@andrewvc do you have thoughts here? I'm fine with changing these, cleaner and not losing any information/context, since the outer button says `Create alert` already.

Reply:
For reference, this change results in a context menu like:

![image](https://user-images.githubusercontent.com/18429259/80978276-d77bb700-8df3-11ea-9b2b-5b7013d8f1fe.png)

Reply:
We might not even need `alert`. Thoughts?